aerospike/requirements.in

```diff
@@ -1 +1 @@
-aerospike==3.10.0; sys_platform != 'win32'
+aerospike==3.10.0; sys_platform != 'win32' and sys_platform != 'darwin'
```

Comment:
We can't since this affects the Agent

Reply:
Hmm, then we won't be able to run env since tox tries to install that package on host system ..

Reply:
modify `tox.ini`?